ome_zarr/writer.py

```diff
@@ -321,6 +321,29 @@ def write_multiscales_metadata(
             axes = _get_valid_axes(axes=axes, fmt=fmt)
             if axes is not None:
                 ndim = len(axes)
+    print(metadata)
```

Comment:
best to use `LOGGER.debug` here if needed instead of `print`

Reply:
removed, forgot to delete before.